src/js/util/headlights.js

```diff
@@ -44,6 +45,7 @@ define(function (require, exports) {
      */
     var logEvent = function (category, subcategory, event) {
         if (global.debug) {
+            log.info("headlights: %s > %s > %s", category, subcategory, event);
```

Comment:
Remove this before merging. See my identical complaint in @shaoshing's PR #3097.